backends/qualcomm/quantizer/observers/param_observer.py

```diff
@@ -0,0 +1,103 @@
+import torch
+from torch.ao.quantization.observer import UniformQuantizationObserverBase
+
+
+class ParamObserver(UniformQuantizationObserverBase):
```

Comment:
This looks great and is generic enough to be added into torch/ao/quantization/observer.py. If possible, please move it there as a follow-up. 

How about renaming it to PerChannelParamObserver() or PerChannelWeightObserver?

Reply:
It's great to see you like it! Change the name and add a TODO for it